static/app/views/releases/detail/commitsAndFiles/commits.tsx

```diff
@@ -24,6 +23,7 @@ import {useRepositories} from 'sentry/utils/useRepositories';
 import {formatVersion} from 'sentry/utils/versions/formatVersion';
 import {ReleaseContext} from 'sentry/views/releases/detail';
 import {ReleaseCommit} from 'sentry/views/releases/detail/commitsAndFiles/releaseCommit';
+import {useReleaseCommits} from 'sentry/views/releases/utils/useReleaseCommits';
```

Comment:
made a sep PR for these changes but they haven't been reviewed yet https://github.com/getsentry/sentry/pull/86614